cmake/Modules/FindComputeCpp.cmake

```diff
@@ -232,13 +238,18 @@ endfunction()
 #  target and sets a dependancy on that new command.
 #
 #  targetName : Name of the target to add a SYCL to.
-#  sourceFile : Source file to be compiled for SYCL.
 #  binaryDir : Intermediate directory to output the integration header.
+#  sourceFiles : Source files to be compiled for SYCL.
 #
-function(add_sycl_to_target targetName sourceFile binaryDir)
+function(add_sycl_to_target targetName binaryDir sourceFiles)
```

Comment:
This interface change breaks all the samples - is it necessary? Does "sourceFiles" represent a variable number of arguments that need to be at the end of the arguments to the function after this change?